apps/backend/.env.example

```
@@ -1,12 +1,2 @@
-SUPABASE_URL=
-SUPABASE_ANON_KEY=
-
-# This was inserted by `prisma init`:
-# Environment variables declared in this file are automatically made available to Prisma.
-# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
-
-# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
-# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
-
 DATABASE_URL=
 DIRECT_URL=
```

```suggestion
DATABASE_URL="postgresql://postgres.lawmtehokpfpvntldokb:[YOUR-PASSWORD]@vez2ckx@aws-0-ap-northeast-1.pooler.supabase.com:6543/postgres"
DIRECT_URL="postgresql://postgres.lawmtehokpfpvntldokb:[YOUR-PASSWORD]@vez2ckx@aws-0-ap-northeast-1.pooler.supabase.com:6543/postgres"
```
パスワードをマスクした状態にしておけば、新たに環境構築する際にスムーズになるかと思ったのですが、いかがでy草加？